frontend/surgeon.js

```diff
@@ -65,6 +68,3 @@ export function nHide (el, n) {
   el = nEl(el)
   return n(v => { el.style.display = v ? 'none' : null })
 }
-export function nShow (el, n) {
```

Comment:
Thanks for removing. This function have been glowing red since i first cloned the repo 😁